demo/examples/single-line-string.js

```diff
@@ -28,6 +28,14 @@ const fields = [
   },
   {
     id: 'single-line-string-4',
+    label: 'Disabled Name',
+    type: 'single-line-string',
+    key: 'readonlyName',
```

Comment:
I know it’s just a demo but do we maybe want to name this `disabledName` to avoid confusion in the demo?